src/uct/api/v2/uct_v2.h

```diff
@@ -1151,6 +1163,23 @@ ucs_status_t uct_ep_connect_to_ep_v2(uct_ep_h ep,
 int uct_ep_is_connected(uct_ep_h ep,
                         const uct_ep_is_connected_params_t *params);
 
+
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Create a device endpoint.
+ *
+ * @param [in]  ep          Endpoint to create a device endpoint from.
+ * @param [in]  params      Parameters for creating a device endpoint.
+ * @param [out] device_ep_p Filled with the created device endpoint.
+ *
+ * @return UCS_OK           Device endpoint created successfully.
+ *         Other            Error codes as defined by @ref ucs_status_t.
+ */
+ucs_status_t uct_ep_create_device_ep(uct_ep_h ep,
```

Comment:
maybe `uct_ep_get_device_ep` -
- there are no destory
- it allow multiple calls return same value